cfgov/locale/es/LC_MESSAGES/django.po

```diff
@@ -1,28 +1,32 @@
 # Spanish translations for cfgov-refresh
-
 #, fuzzy
 msgid ""
 msgstr ""
-"POT-Creation-Date: 2017-08-30 20:30-0400\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-11-09 17:22-0500\n"
 "Language: Spanish\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: ask_cfpb/models/pages.py:590
+#: ask_cfpb/models/pages.py:603 ask_cfpb/tests/models/test_pages.py:712
 msgid "redirected but not live"
 msgstr ""
 
-#: ask_cfpb/models/pages.py:592
+#: ask_cfpb/models/pages.py:605 ask_cfpb/tests/models/test_pages.py:715
```

Comment:
is this supposed to be the tests dir?

Reply:
No idea! :) I'm very new to using Django translation.

Reply:
Ultimately this shouldn't matter at all, but I am curious why it picked up these strings in those files this time but not before.